src/lib/config/pools.ts

```diff
@@ -0,0 +1,6 @@
+import { Pools } from '@/types/pools';
+import { configService } from '@/services/config/config.service';
+
+const pools: Pools = configService.network.pools;
```

Comment:
I was thinking it's config data, not a constant, as we export config in lowercase rather than `CONFIG` too.

Reply:
I might actually call it `poolsConfig` everywhere, just pools is a little too ambiguous.

Reply:
Actually it is constant for the lifetime of the program, like we're not changing any of this config post compilation, so it can make sense for it to be `POOLS`.

Reply:
I would vote for networkPools (or NETWORK_POOLS) instead. 

As I new developer joining the project I would probably expect pools to be a list of pools that we fetch from somewhere. However, IMO `networkPools` is clearer. 

Also agree that uppercase (POOLS or NETWORK_POOLS) implicitly gives it "static" nuance.